test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -363,6 +363,8 @@ $redirectTests = @(
 
     @{redirectType = 'TemporaryRedirect'; redirectedMethod = 'GET'}
     @{redirectType = 'RedirectKeepVerb'; redirectedMethod = 'GET'} # Synonym for TemporaryRedirect
+
+    @{redirectType = 'relative'; redirectedMethod = 'GET'}
```

Comment:
are the existing tests enough to verify that you are making the uri absolute?

Reply:
I'm not sure I understand what you are asking. But, if I were to role back the code change and leave, the test would fail.This one would be hard to verify directly without some kind of debug or test hook. Instead, I decided to test the behavior.

Since this bug identified awhile ago, I had WebListener set to always return absolute URIs for tests. I had to add an exception to that behavior now that it is fixed with the new `type` value.

Reply:
This answers my question well enough